### PR TITLE
Ldanzinger/pri updates

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#-------------------------------------------------
+#  Copyright 2016 ESRI
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri]: 2017 copyright for new files.

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#
+#  You may freely redistribute and use this sample code, with or
+#  without modification, provided you include the original copyright
+#  notice and use restrictions.
+#
+#  See the Sample code usage restrictions document for further information.
+#-------------------------------------------------
+
+contains(QMAKE_HOST.os, Windows):{
+  iniPath = "C:\\ProgramData\\EsriRuntimeQt\\ArcGIS Runtime SDK for Qt $${ARCGIS_RUNTIME_VERSION}.ini"
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri]: Can we use `ALLUSERSPROFILE` environment variable instead?

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri (hunk 1 above)
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri]: Same for others below.

File: ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri (hunk 2 above)
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/arcgisruntime.pri]: Same for others below.

**Comment by ldanzinger** [comment]
good call @michael-tims. updated

**Pull request merged**